site/docs/v1.2.0/annotations.md

```diff
@@ -74,5 +58,5 @@ A [Kubernetes Service][9] maps to an [Envoy Cluster][10]. Envoy clusters have ma
 [12]: https://www.envoyproxy.io/docs/envoy/v1.11.2/api-v2/api/v2/cluster/circuit_breaker.proto#envoy-api-field-cluster-circuitbreakers-thresholds-max-pending-requests
 [13]: https://www.envoyproxy.io/docs/envoy/v1.11.2/api-v2/api/v2/cluster/circuit_breaker.proto#envoy-api-field-cluster-circuitbreakers-thresholds-max-requests
 [14]: https://www.envoyproxy.io/docs/envoy/v1.11.2/api-v2/api/v2/cluster/circuit_breaker.proto#envoy-api-field-cluster-circuitbreakers-thresholds-max-retries
-[15]: {% link docs/v1.2.0/ingressroute.md %}
+[15]: {% link docs/master/ingressroute.md %}
```

Comment:
This one should still link to the 1.2.0 docs since it is in the versioned tree.

Reply:
Ahh yup. I'm going to change it so we don't need to manage versions like this. We did that with a number of links.